debian/set_wan_led.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+LED_PATH="/sys/class/leds/wan-led"
+
+if [ -d /sys/class/net/enp1s0 ]; then
```

Comment:
```suggestion
if [[ -d /sys/class/net/wan ]]; then
```
用bash就不要用`[ ]`了，一律用`[[ ]]`，更安全
另外优先测试wan，如果一个设备上同时有wan和enp1s0那么更大概率是wan作为wan口